tests/parametric/test_otel_span_methods.py

```diff
@@ -484,29 +484,72 @@ def test_otel_span_reserved_attributes_overrides(self, test_agent, test_library)
         assert "analytics.event" not in span["meta"]
 
     @missing_feature(context.library < "java@1.25.0", reason="Implemented in 1.25.0")
-    @missing_feature(context.library == "nodejs", reason="Not implemented")
     @missing_feature(context.library == "dotnet", reason=".NET dosn't treat 'something-else' as a valid input here.")
```

Comment:
This can be un-skipped for .NET as `something-else` is no longer in this test and we pass all of them.

Reply:
```suggestion
```